pages/architecture/session_tokens_rfc.rst

```diff
@@ -0,0 +1,152 @@
+.. _nuts-documentation-session-tokens:
+
+RFC: Session Tokens
+###################
+
+Session tokens are tokens which should be provided along an API request which refer
+to contextual information of the request.
+A session is scoped to an actor, subject and custodian. Optionally a reference
+to a specific consent record can be provided.
+
+Motivation
+**********
+
+Originally the plan was to only provide the Actor's
+:ref:`nuts-documentation-login-contracts` along the API request, assuming this
+was sufficient. However we encountered situations where more information was
+needed.
+
+Size
+####
+
+The more attributes are requested to sign the IMRA login contract, the bigger
+the resulting JSON hash gets. So big that when base64 encoded and signed as a
```

Comment:
'JSON hash' is this correct, isn't it just JSON (since hashes are generally fixed size)?

Reply:
As far as I know the `hash` part of a `JSON hash` is refering to the JSON object or key-value store. So what I try to tell here is that the amount of data gets so large it no longer fits in the header.
What do you mean by fixed size?

Reply:
Nevermind. I thought you meant a (fixed size) hash of a JSON struct, but I understand you mean the JSON having a map-like structure.